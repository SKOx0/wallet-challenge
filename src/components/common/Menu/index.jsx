import React from 'react';
import { Menu, MenuLabel, MenuList } from 'bloomer';
import { NavLink } from 'react-router-dom';

const MenuComponent = () => (
  <Menu >
    <MenuList>
      <li><NavLink exact to="/home" activeClassName="is-active">Inicio</NavLink></li>
    </MenuList>
    <MenuLabel>Moedas</MenuLabel>
    <MenuList>
      <li><NavLink to="/home/bitcoin" activeClassName="is-active">Bitcoin</NavLink></li>
      <li><NavLink to="/home/brita" activeClassName="is-active">Brita</NavLink></li>
    </MenuList>
  </Menu>
);

export default MenuComponent;
