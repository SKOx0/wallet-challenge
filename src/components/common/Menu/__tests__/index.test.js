import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Menu from '../index';

describe('<Menu />', () => {
  it('deve ser igual ao snapshot', () => {
    const renderedComponent = shallow(<Menu />);

    expect(toJson(renderedComponent)).toMatchSnapshot();
  });
});
