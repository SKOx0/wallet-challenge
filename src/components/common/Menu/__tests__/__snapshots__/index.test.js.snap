// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Menu /> deve ser igual ao snapshot 1`] = `
<withHelpersModifiers(Menu)>
  <withHelpersModifiers(MenuList)>
    <li>
      <NavLink
        activeClassName="is-active"
        aria-current="page"
        exact={true}
        to="/home"
      >
        Inicio
      </NavLink>
    </li>
  </withHelpersModifiers(MenuList)>
  <withHelpersModifiers(MenuLabel)>
    Moedas
  </withHelpersModifiers(MenuLabel)>
  <withHelpersModifiers(MenuList)>
    <li>
      <NavLink
        activeClassName="is-active"
        aria-current="page"
        to="/home/bitcoin"
      >
        Bitcoin
      </NavLink>
    </li>
    <li>
      <NavLink
        activeClassName="is-active"
        aria-current="page"
        to="/home/brita"
      >
        Brita
      </NavLink>
    </li>
    <li>
      <NavLink
        activeClassName="is-active"
        aria-current="page"
        to="/home/transacoes"
      >
        Transações
      </NavLink>
    </li>
  </withHelpersModifiers(MenuList)>
</withHelpersModifiers(Menu)>
`;
